src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -173,6 +174,11 @@
                             currentFieldGroup.values.push(currentField.id);
                         });
                     }
+                    //special case so Amount object is always visible
+                    if (sObjectName === 'Opportunity') {
+                        currentFieldGroup.requiredOptions.push('Opportunity.npsp__Donation_Amount__c');
```

Comment:
Is there a way to be clever about namespace here so we don't have to repeat these?

Reply:
Will have to give some additional thought to this, as it's not immediately clear with the pattern the best way to do this.

Reply:
@randi274 can we move this logic to the Apex Controller (BGE_ConfigurationWizard_CTRL). So we can get this required options from the Controller. That way, we can reuse the Namespace Utility classes to get the namespace?

Reply:
@kofijohnson and @lmeerkatz I changed this up a little bit, since it's such a small change. Is this ok, or would a larger refactor still be better?